src/model/openthaigpt_pretraining_model/models/GPTJ_TH_tokenizer/tokenizer.py

```diff
@@ -0,0 +1,38 @@
+from transformers import PreTrainedTokenizerFast
+from constants import GPTJ_TOKEN_DIR, NEW_TOKEN_DIR
+from typing import Any, List, Union
+
+
+class GPTJToken:
+    def __init__(self):
+        self.tokenizer = PreTrainedTokenizerFast(tokenizer_file=GPTJ_TOKEN_DIR)
+
+    def tokenize(self, x: str) -> List[Any]:
+        return self.tokenizer.tokenize(x)
+
+    def encode(self, x: str) -> List[Any]:
+        return self.tokenizer.encode(x)
+
+    def decode(self, x: Union[int, List[int]]) -> str:
+        return self.tokenizer.decode(x)
+
+
+class MergedToken:
+    def __init__(self):
+        self.tokenizer = PreTrainedTokenizerFast(tokenizer_file=NEW_TOKEN_DIR)
+
+    def tokenize(self, x: str) -> List[Any]:
+        return self.tokenizer.tokenize(x)
+
+    def encode(self, x: str) -> List[Any]:
+        return self.tokenizer.encode(x)
+
+    def decode(self, x: Union[int, List[int]]) -> str:
+        return self.tokenizer.decode(x)
+
+
+text = "รายละเอียดและหลักเกณฑ์การคัดเลือก AI Startup Incubation by AIEAT"
```

Comment:
You should put the code below this to src/model/scripts. Then you can create you own folder. After you move it, please add README.md and describe what you do.

Reply:
I' ve commented lines below and move them to the test case in tests/model and I've move the token merging script to src/model/scripts and also add README.md file in that folder.